specs/opcode/46CHAINID.md

```diff
@@ -0,0 +1,36 @@
+# CHAINID opcode
+
+## Procedure
+
+The `CHAINID` opcode gets the chain id where a transaction is meant executed at.
+
+## EVM behaviour
+
+The `CHAINID` opcode loads a 256-bit value from the Tx context (the chain id), and pushes it to the stack.
```

Comment:
```suggestion
The `CHAINID` opcode loads a 256-bit value from the chain configuration (the chain id), and pushes it to the stack.
```

Reply:
Ohh dammit! Since I started the specs as if the value was taken from the `tx_table`, I forgot to change this once we decided to move it to the block!!

Reply:
Eventhough the implementation loads the chain_id from the block table, I think it's good to use the definition from Han (that is, "from the chain configuration"), and then explain that in our case we add this value in the block table for convenience.